README.md

```diff
@@ -18,6 +18,30 @@ git clone https://github.com/redhat-et/copilot-ops.git
 # Build the binary
 make build
 ```
+To install the command line tool, complete the following steps:
+
+```console
+# Create bin directory
+mkdir -p ${HOME}/bin
```

Comment:
I would prefer to keep the commands to set up a per-user bin-dir in a separate script than copying copilot-ops to it and testing it, just because it's a different flow and some users would skip one or the other.